sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -157,6 +158,52 @@ def read(  # pylint:disable=docstring-missing-param
         )
         return self._properties
 
+    @overload
+    def create_container(  # pylint:disable=docstring-missing-param
```

Comment:
Would be worth adding a snippet/sample somewhere showing how this can be used with get client:

```python
container = client.create_container(..., return_headers=True)
container_client = client.get_container_client(container)
```